service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/StandardDirectionsOrderIssuedEventHandler.java

```diff
@@ -62,16 +63,22 @@ public void notifyLocalAuthorityOfIssuedStandardDirectionsOrder(StandardDirectio
 
     @EventListener
     public void notifyAllocatedJudgeOfIssuedStandardDirectionsOrder(StandardDirectionsOrderIssuedEvent event) {
-        if (featureToggleService.isAllocatedJudgeNotificationEnabled(SDO)) {
```

Comment:
Shouldn't this still be toggled off?

Reply:
Whoops did not mean to do that. Strange how unit tests did not fail though. I'll look into why it did not fail.